socorro/processor/mozilla_processor_2015.py

```diff
@@ -63,6 +63,7 @@
         "socorro.processor.signature_utilities.SignatureRunWatchDog, "
         "socorro.processor.signature_utilities.SignatureIPCChannelError, "
         "socorro.processor.signature_utilities.SignatureIPCMessageName, "
+        "socorro.processor.signature_utilities.SigTrim, "
```

Comment:
I think it makes sense to strip it before truncating so the extra spaces don't affect truncation. Having said that, maybe `SigTrunc` should take the signature, strip it and then truncate it and make sure it's stripped?